Adapt repository to deploy Wazuh 4.4.0

The repository needs to be adapted to deploy Wazuh `4.4.0` using branch `4.4`.
The required change so far is to set `/etc/wazuh-indexer/opensearch-security` to the variable `indexer_sec_plugin_conf_path` for the indexer role.

After changing the variable the pipeline failed with the following error when running the `securityadmin` script:

```TXT
09:51:55  Security Admin v7
09:51:55  Will connect to 10.0.2.96:9300 ... done
09:51:55  ERR: An unexpected RuntimeException occured: error while performing request
09:51:55  Trace:
09:51:55  java.lang.RuntimeException: error while performing request
09:51:55  	at org.opensearch.client.RestClient.extractAndWrapCause(RestClient.java:963)
09:51:55  	at org.opensearch.client.RestClient.performRequest(RestClient.java:332)
09:51:55  	at org.opensearch.client.RestClient.performRequest(RestClient.java:320)
09:51:55  	at org.opensearch.security.tools.SecurityAdmin.execute(SecurityAdmin.java:462)
09:51:55  	at org.opensearch.security.tools.SecurityAdmin.main(SecurityAdmin.java:159)
09:51:55  Caused by: org.apache.http.ProtocolException: Not a valid protocol version: This is not an HTTP port
```

The error will be investigated. Related issue in Opensearch Project:
- https://github.com/opensearch-project/documentation-website/issues/695

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,12 @@ All notable changes to this project will be documented in this file.
 
 - Update to [Wazuh v4.4.0](https://github.com/wazuh/wazuh/blob/v4.4.0/CHANGELOG.md#v440)
 
+## [v4.3.9]
+
+### Added
+
+- Update to [Wazuh v4.3.9](https://github.com/wazuh/wazuh/blob/v4.3.9/CHANGELOG.md#v439)
+
 ## [v4.3.8]
 
 ### Added

--- a/README.md
+++ b/README.md
@@ -16,6 +16,7 @@ These playbooks install and configure Wazuh agent, manager and indexer and dashb
 | Wazuh version | Elastic | ODFE   |
 |---------------|---------|--------|
 | v4.4.0        |         |        |
+| v4.3.9        |         |        |
 | v4.3.8        |         |        |
 | v4.3.7        |         |        |
 | v4.3.6        |         |        |

--- a/roles/wazuh/ansible-wazuh-agent/tasks/Debian.yml
+++ b/roles/wazuh/ansible-wazuh-agent/tasks/Debian.yml
@@ -1,14 +1,22 @@
 ---
-- name: Debian/Ubuntu | Install apt-transport-https, ca-certificates, gnupg and acl
+- name: Debian/Ubuntu | Install ca-certificates and gnupg
   apt:
     name:
-      - apt-transport-https
       - ca-certificates
-      - acl
       - gnupg
     state: present
   register: wazuh_agent_ca_package_install
   until: wazuh_agent_ca_package_install is succeeded
+
+- name: Debian/Ubuntu | Install apt-transport-https and acl
+  apt:
+    name:
+      - apt-transport-https
+      - acl
+    state: present
+  register: wazuh_agent_ca_package_install
+  until: wazuh_agent_ca_package_install is succeeded
+  when: not (ansible_distribution == "Debian" and ansible_distribution_major_version in ['11'])
 
 - name: Debian/Ubuntu | Installing Wazuh repository key (Ubuntu 14)
   become: true

--- a/roles/wazuh/ansible-wazuh-agent/templates/var-ossec-etc-ossec-agent.conf.j2
+++ b/roles/wazuh/ansible-wazuh-agent/templates/var-ossec-etc-ossec-agent.conf.j2
@@ -260,7 +260,7 @@
     {% endif %}
 
     <!-- Directories to check  (perform all possible verifications) -->
-    {% if wazuh_agent_config.syscheck.win_directories is defined and ansible_system == "Windows" %}
+    {% if wazuh_agent_config.syscheck.win_directories is defined and ansible_os_family == "Windows" %}
     {% for directory in wazuh_agent_config.syscheck.win_directories %}
     <directories {{ directory.checks }}>{{ directory.dirs }}</directories>
     {% endfor %}
@@ -280,7 +280,7 @@
     {% endfor %}
     {% endif %}
 
-    {% if wazuh_agent_config.syscheck.ignore is defined and ansible_system == "Windows" %}
+    {% if wazuh_agent_config.syscheck.ignore is defined and ansible_os_family == "Windows" %}
     {% for ignore in wazuh_agent_config.syscheck.ignore_win %}
     <ignore type="sregex">{{ ignore }}</ignore>
     {% endfor %}

--- a/roles/wazuh/wazuh-dashboard/defaults/main.yml
+++ b/roles/wazuh/wazuh-dashboard/defaults/main.yml
@@ -11,7 +11,7 @@ dashboard_server_name: "dashboard"
 wazuh_version: 4.4.0
 indexer_cluster_nodes:
   - 127.0.0.1
-  
+
 # The Wazuh dashboard package repository
 dashboard_version: "4.4.0"
 

--- a/roles/wazuh/wazuh-dashboard/vars/debian.yml
+++ b/roles/wazuh/wazuh-dashboard/vars/debian.yml
@@ -1,3 +1,2 @@
 ---
-
 dashboard_version: 4.4.0

--- a/roles/wazuh/wazuh-indexer/defaults/main.yml
+++ b/roles/wazuh/wazuh-indexer/defaults/main.yml
@@ -28,7 +28,7 @@ domain_name: wazuh.com
 
 indexer_sec_plugin_conf_path: /etc/wazuh-indexer/opensearch-security
 indexer_sec_plugin_tools_path: /usr/share/wazuh-indexer/plugins/opensearch-security/tools
-indexer_conf_path: /etc/wazuh-indexer/
+indexer_conf_path: /etc/wazuh-indexer
 indexer_index_path: /var/lib/wazuh-indexer/
 
 # Security password

--- a/roles/wazuh/wazuh-indexer/defaults/main.yml
+++ b/roles/wazuh/wazuh-indexer/defaults/main.yml
@@ -26,7 +26,7 @@ minimum_master_nodes: 2
 # Example es1.example.com, es2.example.com
 domain_name: wazuh.com
 
-indexer_sec_plugin_conf_path: /usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig
+indexer_sec_plugin_conf_path: /etc/wazuh-indexer/opensearch-security
 indexer_sec_plugin_tools_path: /usr/share/wazuh-indexer/plugins/opensearch-security/tools
 indexer_conf_path: /etc/wazuh-indexer/
 indexer_index_path: /var/lib/wazuh-indexer/

--- a/roles/wazuh/wazuh-indexer/tasks/security_actions.yml
+++ b/roles/wazuh/wazuh-indexer/tasks/security_actions.yml
@@ -83,7 +83,7 @@
       JAVA_HOME=/usr/share/wazuh-indexer/jdk
       {{ indexer_sec_plugin_tools_path }}/securityadmin.sh
       -cd {{ indexer_sec_plugin_conf_path }}/
-      -icl -p 9300 -cd {{ indexer_sec_plugin_conf_path }}/
+      -icl -p 9200 -cd {{ indexer_sec_plugin_conf_path }}/
       -nhnv
       -cacert {{ indexer_conf_path }}/certs/root-ca.pem
       -cert {{ indexer_conf_path }}/certs/admin.pem

